package.json

```diff
@@ -64,8 +66,6 @@
         "@typescript-eslint/parser": "5.39.0",
         "commitlint": "^17.0.3",
         "crawlee": "^3.0.0",
-        "playwright": "*",
-        "puppeteer": "*",
```

Comment:
I think these should stay here, no? They're used in the e2e tests.

Reply:
I think it's gonna work, since they're still in the lockfile, but they're imported from the scraper source code, which is then imported dynamically through here: https://github.com/apify/apify-sdk-js/blob/6d1af7916e576e30abd9a494881540f1598447a2/test/e2e/tools.mjs#L58

But I don't know how it works with the workspaces, if you reference a workspace which has them listed as dependencies, maybe you don't need to list them as your own dependencies? I guess it's fine in that case then.

Reply:
Test passed so it should be fine.